tests/scorer/test_multiscorer.py

```diff
@@ -63,7 +63,7 @@ def test_single_scorer() -> None:
     )
 
     # normal eval
-    log = eval(tasks=task, model="mockllm/model")[0]
```

Comment:
These should be switched back to mockllm/model

Reply:
fixed! 🫡